playwright/features/itemDetailPageElements.feature

```diff
@@ -20,4 +24,9 @@ Feature: Item Detail Page Elements
         And the "item all editions heading" should be displayed
         And the "items currently available online toggle text" should be displayed
         And the "items currently available online toggle" should be displayed
-        And the "second item edition" should be displayed
\ No newline at end of file
+        And the "second item edition" should be displayed
+
+    Scenario: As a user when I click on "show only intems currently available online" button, only online available books should be displayed
+        Given I go to the "item detail" page
+        Then I double click the "items currently available online toggle"
+        And the "first read online button" should be displayed
```

Comment:
Noting that we discussed how the "first read online button" would look for the Read Online button under Featured Edition button versus the button under All Editions.